### PR TITLE
Fix missing description and http --> https

### PR DESCRIPTION
**Pull request by velkovb**

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306  @tseward

Peer review needed for this PR. Thanks.

File: docset/windows/dhcpserver/Get-DhcpServerVersion.md [hunk 1]
```diff
@@ -50,8 +50,8 @@ Use this parameter to run commands that take a long time to complete.
 The cmdlet immediately returns an object that represents the job and then displays the command prompt.
 You can continue to work in the session while the job completes.
 To manage the job, use the `*-Job` cmdlets.
-To get the job results, use the [Receive-Job](http://go.microsoft.com/fwlink/?LinkID=113372) cmdlet. 
-For more information about Windows PowerShell® background jobs, see [about_Jobs](http://go.microsoft.com/fwlink/?LinkID=113251).
+To get the job results, use the [Receive-Job](https://go.microsoft.com/fwlink/?LinkID=113372) cmdlet. 
+For more information about Windows PowerShell® background jobs, see [about_Jobs](https://go.microsoft.com/fwlink/?LinkID=113251).
```
Comment (JohanFreelancer9) [inline docset/windows/dhcpserver/Get-DhcpServerVersion.md]:
```suggestion
For more information about Windows PowerShell background jobs, see [about_Jobs](https://go.microsoft.com/fwlink/?LinkID=113251).
```

File: docset/winserver2012r2-ps/dhcpserver/Get-DhcpServerVersion.md [hunk 2]
```diff
@@ -44,7 +44,13 @@ This example gets the major and minor version of the DHCP server service running
...
 
 ### -AsJob
-ps_cimcommon_asjob
+Runs the cmdlet as a background job.
+Use this parameter to run commands that take a long time to complete. 
+The cmdlet immediately returns an object that represents the job and then displays the command prompt.
+You can continue to work in the session while the job completes.
+To manage the job, use the `*-Job` cmdlets.
+To get the job results, use the [Receive-Job](https://go.microsoft.com/fwlink/?LinkID=113372) cmdlet. 
+For more information about Windows PowerShell® background jobs, see [about_Jobs](https://go.microsoft.com/fwlink/?LinkID=113251).
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/dhcpserver/Get-DhcpServerVersion.md]:
```suggestion
For more information about Windows PowerShell background jobs, see [about_Jobs](https://go.microsoft.com/fwlink/?LinkID=113251).
```

**Pull request merged**